CodeGen/Generators/UnitsNetGen/QuantityGenerator.cs

```diff
@@ -256,6 +261,12 @@ private void GenerateStaticProperties()
             }
 
             Writer.WL($@"
+
+        /// <summary>
+        /// The per-culture abbreviations. To add a custom default abbreviation, add to the beginning of the list.
+        /// </summary>
+        public static IDictionary<(CultureInfo Culture, {_unitEnumName} Unit), List<string>> Abbreviations {{ get; }}
```

Comment:
Before even reading the rest, shouldn't this rather sit in `QuantityInfo`?
- This is where units and other data for a quantity is stored
- Reusable/extensible when working with `IQuantity`
- Reduce member count per quantity (binary size)

Reply:
Reading more, yes, `GetAbbreviations()` etc seems to me would sit nicely in `QuantityInfo`.

Reply:
The syntax would only be slightly longer `Length.Info.GetAbbreviation()`.

Reply:
@angularsen I like that. Let me give it a shot here